### ISSUE TITLE
Move language switch or emoji key to the right of the spacebar

### ISSUE THREAD

**Issue by Godeps2891** [openboard-team/openboard#621]
Enabling both the emoji key and the language switch key OpenBoard very awkwardly moves the spacebar around when changing from letters to symbols even though both pages have the same amount of keys.

![image](https://user-images.githubusercontent.com/54289108/172910667-485f5805-ddc9-4732-9904-3f517833c903.png)
![image](https://user-images.githubusercontent.com/54289108/172910678-4aee588f-8472-4a32-919a-df3ee6eaa260.png)

**Comment by MajeurAndroid** [comment]
You're right, I think we should prevent the enabling of both emoji and languages keys, as it is done in gboard.

**Comment by Godeps2891** [comment]
I'm not sure i agree on that solution if i understood correctly, i use both keys frequently and allowing only one to be enabled at a time would still result in changing sizes/positions of keys when switching between letters and symbols

my proposal would have been the following layout when both are enabled
![image](https://user-images.githubusercontent.com/54289108/178127796-adc09bfd-ad25-4285-8f23-0bc15baa879b.png)
